cmd/watch.go

```diff
@@ -83,46 +83,56 @@ func init() {
 // 	return
 // }
 
-// Parses the Event FileInfo and returns an EventInfo slice or error
-func EventSrc(e fsnotify.Event) func() ([]EventInfo, error) {
-	ep := e.Name
-	// nil by default
-	var eo error
-	ein := EventInfo{}
-	einfo := make([]EventInfo, 0)
-	if !path.IsAbs(e.Name) {
-		pwd, err := os.Getwd()
-		if err != nil {
-			// log.Fatal(err)
-			eo = errors.New("[EventSrc(event)] os.Getwd failed..")
-		}
-		ep = path.Join(pwd, e.Name)
+type FsEventOperations interface {
+	EventPath(path string) (string, error)
+	FsInfo(path string) (os.FileInfo, error)
+}
+
+type FsEventOps struct {
```

Comment:
In unit test you have to re-implement the methods `EventPath` and `FsInfo` on a `FsEventOpsTest` without calling the `os` package just return some test values you need.